ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/signatures/AggregatingSignatureVerificationService.java

```diff
@@ -65,11 +65,13 @@
 
     this.batchSignatureTasks = new ArrayBlockingQueue<>(queueCapacity);
     this.minBatchSizeToSplit = minBatchSizeToSplit;
-    metricsSystem.createGauge(
-        TekuMetricCategory.EXECUTOR,
-        "signature_verifications_queue_size",
-        "Tracks number of signatures waiting to be batch verified",
-        this::getQueueSize);
+    metricsSystem
+        .createLabelledGauge(
+            TekuMetricCategory.EXECUTOR,
+            "queues",
+            "Current queues size for this executor",
+            "signature_verifications_queue_size")
+        .labels(this::getQueueSize, "signature_verifications_queue_size");
```

Comment:
Not sure why we've switched this one to use a labelled gauge with only one label. I think it makes most sense for now to leave this as-is - at some point in the future maybe we have a completely generic queue size metric but this ticket was just aimed at the `MetricTrackingExecutorFactory` ones which are all in the same place and can reasonably use the same gauge with different labels.